qa/zcash/smoke_tests.py

```diff
@@ -337,10 +337,7 @@ def transaction_chain(zcash):
         results['5c'] = False
 
     # Set up beginning and end of the chain
-    print('#')
-    print('# Initialising transaction chain')
-    print('#')
-    print()
+    print('\n#\n# Initialising transaction chain\n#\n')
```

Comment:
This unnecessarily hard-codes newlines, which both makes the formatting harder to see in the script, and make it incompatible with `\r\n` environments. Please remove these kinds of changes from the entire PR.